### PR TITLE
Typos: Fix some typos in the so

### PR DESCRIPTION
**Pull request by dungxibo123**
### Just a fixing typos

 #xxx

### How is this PR tested?

- [ ] Existing unit/integration tests
- [ ] New unit/integration tests
- [ x ] Manual tests

### Does this PR require documentation update?

- [ ] No. You can skip the rest of this section.
- [ x ] Yes. I've updated:
  - [ ] Examples
  - [ ] API references
  - [ x ] Instructions

#### Is this a user-facing change?

- [ x ] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

#### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/gateway`: AI Gateway service, Gateway client APIs, third-party Gateway integrations
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ x ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

#### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ x ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Comment by B-Step62** [comment]
@dungxibo123 Thank you for these fixes!

1. Could you run `pre-commit run --all-files` to run linters and formatters? 
2. Could you add sign off to each commit (required by CI check)? You can do this by `git rebase -i HEAD~3` and apply `git commit -s --amend --no-edit` for each commit.

**Comment by dungxibo123** [comment]
@B-Step62 sure, I will

File: mlflow/diviner/__init__.py [hunk 1]
```diff
@@ -136,7 +136,7 @@ def save_model(
                      .. Note:: Experimental: This parameter may change or be removed in a future
                                              release without warning.
 
-    :param kwargs: Optional configurations for Spark DataFrame storage iff the model has
+    :param kwargs: Optional configurations for Spark DataFrame storage if the model has
```
Comment (BenWilson2) [inline mlflow/diviner/__init__.py:139]: This isn't a typo.

**Comment by BenWilson2** [comment]
None of these are typos. Please see https://en.wikipedia.org/wiki/If_and_only_if for guidance on what these terms are referring to. 
Thank you for the intention, though! If you see other things that you'd like to contribute to, don't hesitate to reach out and pick up one of our good first issues!

**Pull request closed without merging**